src/main/java/se/iths/springbootgroupproject/entities/User.java

```diff
@@ -67,6 +66,20 @@ public void setId(Long id) {
         this.id = id;
     }
 
+    public void setFullName(String fullName) {
+        String[] firstAndLastName = fullName.split(" ");
```

Comment:
This can be simplified by adding
	 `.trim().split("\\s+”);`
instead of 
	`.split(“ ”);`
It also handles leading and trailing spaces. Example “       Efva   Attling      “ Then you need to trim() first before split() it into first- and lastname.